depthai_demo.py

```diff
@@ -226,6 +226,9 @@ def __init__(self, source, model_dir=None, model_name=None):
         if conf.args.cnn_input_size:
             self.input_size = tuple(map(int, conf.args.cnn_input_size.split('x')))
 
+        # Count objects detected on the frame
+        self.count = conf.getCountLabel(self)
```

Comment:
Can we rename the self.count and conf.args.count to `count_label`? 

The cli param can stay the same, adding `dest` flag to argparse should make it work, it's only a matter of knowing that does a count variable mean